### PR TITLE
Update Expecto version constraint

### PR DESCRIPTION
**Pull request by slang25**
This is the follow up to #17 and will need rebasing

### CODE REVIEW

**Comment by Alxandr** [comment]
Can we rather just close #17 and just have this one? Then no rebasing is needed either. I don't really see the need to release a new version which is tied to the 7->8) version range.

**Comment by slang25** [comment]
@Alxandr good point 👍 I had it in my head it would prevent people from updating expecto without updating the test adaptor. However that only protects people that install this new version, which makes no sense if we immediately superseed it.

**Comment by Alxandr** [comment]
My travis setup is broken. I need to look into that :-/. I'll see if I can fix it this afternoon.

**Comment by slang25** [comment]
Is it the triggering that's failing, or the build itself? I'm happy to help investigate the build if there's an error.

**Comment by Alxandr** [comment]
I've tried migrating from travis.org to travis.com. It's just stuck in limbo 😛. So just config on my end I'm afraid.

File: .appveyor.yml [hunk 1]
```diff
@@ -27,6 +26,10 @@ before_build:
 build_script:
   - cmd: dotnet build build
 
+test_script:
+  - cmd: dotnet test --no-build --logger "trx;LogFileName=testresult.xml" test/Sample.Test/Sample.Test.fsproj
```
Comment (Alxandr) [inline .appveyor.yml]: In theory, the build script should also delegate testing properly. Can you test if this works?

```suggestion
  - cmd: dotnet test --no-build --logger "trx;LogFileName=testresult.xml" build
```

Comment (Alxandr) [reply .appveyor.yml]: Seems that VSTest has changed. I probably need to update my build script/project. I can look at that later if I remember 😛.

It's not too terribly important in this project, cause I only have one test project anyways.

Comment (slang25) [reply .appveyor.yml]: Oh wait, I've just figured it out, it's bundled in the build step. I'll push a fix in a moment.

Comment (slang25) [reply .appveyor.yml]: There we go, I think that's what we want now

**Comment by Alxandr** [comment]
Great. Thanks a lot. Now I just need to figure out how I did releasing on this project 😛

**Comment by slang25** [comment]
Thanks for the quick response. It looks like you push a tag with desired version number and the build script + AppVeyor does the right thing and pushes it to nuget.org

**Comment by Alxandr** [comment]
I know. But I have a script that does the versioning (and tagging). I forgot to format the message of the squash commit correctly though 🤦‍♂️ . Pushed a manual tag though. Should be out as 0.6.0 shortly.

**Comment by Alxandr** [comment]
Pushed to NuGet and validating atm.

**Pull request merged**